src/Microsoft.Identity.Client/Platforms/winrt/DispatcherTaskExtensions.cs

```diff
@@ -37,18 +37,21 @@ internal static class DispatcherTaskExtensions
             Func<Task<T>> func, CoreDispatcherPriority priority = CoreDispatcherPriority.High)
         {
             var taskCompletionSource = new TaskCompletionSource<T>();
-            await dispatcher.RunAsync(priority, async () =>
+
+            async void AgileCallback()
```

Comment:
>AgileCallback [](start = 23, length = 13)

why this name?

Reply:
@henrik-me auto suggestion by resharper. I couldn't think of anything much more meaningful given the context. perhaps TaskCallback is better?

Reply:
thoughts: taskCompletionDelegate/taskCompletionCallback

---
In reply to: [173580599](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/536#discussion_r173580599) [](ancestors = 173580599)